include/bluetooth/hci.h

```diff
@@ -1532,12 +1579,52 @@ struct bt_hci_rp_le_remove_iso_path {
 } __packed;
 
 #define BT_HCI_OP_LE_ISO_TRANSMIT_TEST          BT_OP(BT_OGF_LE, 0x0070)
+struct bt_hci_cp_le_iso_transmit_test {
+	uint16_t handle;
+	uint8_t  payload_type;
+} __packed;
+
+struct bt_hci_rp_le_iso_transmit_test {
+	uint8_t  status;
+	uint16_t handle;
+} __packed;
 
 #define BT_HCI_OP_LE_ISO_RECEIVE_TEST           BT_OP(BT_OGF_LE, 0x0071)
+struct bt_hci_cp_le_iso_receive_test {
+	uint16_t handle;
+	uint8_t  payload_type;
+} __packed;
+
+struct bt_hci_rp_le_iso_receive_test {
+	uint8_t  status;
+	uint16_t handle;
+} __packed;
 
 #define BT_HCI_OP_LE_ISO_READ_TEST_COUNTERS     BT_OP(BT_OGF_LE, 0x0072)
+struct bt_hci_cp_le_read_test_counters {
+	uint16_t handle;
+} __packed;
+
+struct bt_hci_rp_le_read_test_counters {
+	uint8_t  status;
+	uint16_t handle;
+	uint32_t received_packets;
```

Comment:
Maybe drop the "_packet" suffix for brevity?

Reply:
I was going by the already existing bt_hci_rp_le_read_iso_link_quality structure that uses the "_packets" terminology

Reply:
changed to _cnt option